CMSgov/mint-app
# [EASI-4169] Mark Primary Point of Contact

# EASI-4169


## Changes and Description

- GQL
- SQL
- Migration
- Resolvers
- Store methods and corresponding SQL
- Unit Tests (Work in Progress)
- Updated Postman endpoints
- Added primary contact resolver extension to possible operational solution


TODO: 
1. Unit tests currently fail due to lack of utility for store-based assignment of possible operational solution contacts. Attempts at using existing dummy data have failed as there does not currently appear to be dummy data.
2. Postman endpoint for setting primary contact needs to be updated to support dynamic assignment of operational solution contacts. I'm not yet sure how to do this, it's the lowest priority thing for this feature so it hasn't been investigated yet.



## How to test this change

- Restart containers
- Check possible operational solution contacts
  - Assert assigned primary contacts correspond with frontend cards
- Postman endpoints
  - Get all possible operational solutions
    - Assert results contain primary contacts and that they are proper
  - Set a primary operational solution contact

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
